tools/docker/build.sh

```diff
@@ -13,6 +13,7 @@ rootdir="${scriptdir%/*/*}"
 . "${rootdir}/tools/functions.sh"
 
 main() {
+    image=${IMAGE-"${DOCKER_REGISTRY}"prplmesh-builder}
```

Comment:
`IMAGE` is a very generic name.
I think `PRPLMESH_BUILDER_IMAGE` is better in this case.
Also, since now the script can use different build images, you should print the image that is used for the build.

Reply:
Personally I would keep the "DOCKER_REGISTRY" part separate from the image name (in other words, always include the registry part).
```suggestion
    image=${IMAGE-prplmesh-builder}
```
You can still build it locally even if it refers to a remote registry.
It makes even more sense to do so since image-build.sh will tag it with the registry name included (could always do `image-build.sh && build.sh` when you want to build prplmesh).